modules/grpc/loki/loki-dest.cpp

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2023 László Várady
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include "loki-dest.hpp"
+#include "loki-worker.hpp"
+
+#include "compat/cpp-start.h"
+#include "logthrdest/logthrdestdrv.h"
+#include "messages.h"
+#include "template/templates.h"
+#include "compat/cpp-end.h"
+
+#include <cstring>
+#include <string>
+#include <sstream>
+
+constexpr const auto DEFAULT_MESSAGE_TEMPLATE = "$ISODATE $HOST $MSGHDR$MSG";
+
+using syslogng::grpc::loki::DestinationDriver;
+
+struct _LokiDestDriver
+{
+  LogThreadedDestDriver super;
+  DestinationDriver *cpp;
+};
+
+DestinationDriver::DestinationDriver(LokiDestDriver *s)
+  : super(s), url("localhost:9096"), timestamp(LM_TS_PROCESSED),
```

Comment:
To me the default port was 9095, and the docs says this, too: https://grafana.com/docs/loki/latest/configure/

```
# gRPC server listen port.
# CLI flag: -server.grpc-listen-port
[grpc_listen_port: <int> | default = 9095]
```

Am I missing something?

Reply:
You are right. There are signs of 9096 in older(?) configs. Fixed.